dev/TabView/TabViewItem.cpp

```diff
@@ -489,6 +489,35 @@ void TabViewItem::OnPointerCaptureLost(winrt::PointerRoutedEventArgs const& args
     RestoreLeftAdjacentTabSeparatorVisibility();
 }
 
+// Note that the ItemsView will handle the left and right arrow keys if we don't do so before it does,
+// so this needs to be handled below the items view. That's why we can't put this in TabView's OnKeyDown.
+void TabViewItem::OnKeyDown(winrt::KeyRoutedEventArgs const& args)
+{
+    if (!args.Handled())
+    {
+        // Alt+Shift+Arrow reorders tabs, so we don't want to handle that combination.
+        auto isAltDown = (winrt::Window::Current().CoreWindow().GetKeyState(winrt::VirtualKey::Menu) & winrt::CoreVirtualKeyStates::Down) == winrt::CoreVirtualKeyStates::Down;
+        auto isShiftDown = (winrt::Window::Current().CoreWindow().GetKeyState(winrt::VirtualKey::Shift) & winrt::CoreVirtualKeyStates::Down) == winrt::CoreVirtualKeyStates::Down;
+
+        if (!isAltDown || !isShiftDown)
```

Comment:
>|| [](http://example.com/codeflow?start=23&length=2)

Should be "&&"?

Reply:
No, || is correct - we only want to not handle the key press if alt *and* shift are both down.  not (alt and shift) = not alt or not shift.

Reply:
So we DO want to handle e.g. the "alt+right" case and just treat it as "right"?
That seems strange to me.

Reply:
Talked offline - Alt+Right is handled by the ListView by just doing XY focus, which we don't want since that skips the close buttons.